src/test/java/net/datafaker/providers/base/AwsTest.java

```diff
@@ -51,4 +51,11 @@ void testRegion() {
         String region = faker.aws().region();
         assertThat(region).matches("^[a-z]{2}-(south|east|north|west|northeast|central|southeast)-\\d$");
     }
+
+    @Test
```

Comment:
Usually we re-run this test multiple times using annotation `@RepeatableTest(10)` instead of just `@Test`.

Reply:
After replacing the list where prefixes are included in the actual list entries in the YAML file I guess there's no sense to make that test repeatable, correct?